java/ql/test/library-tests/literals/floatLiterals/floatLiterals.ql

```diff
@@ -1,4 +1,8 @@
 import semmle.code.java.Expr
 
+class SrcFloatingPointLiteral extends FloatLiteral {
+  SrcFloatingPointLiteral() { this.getCompilationUnit().fromSource() }
+}
+
 from FloatLiteral lit
```

Comment:
Did you intend to change this to be `SrcFloatingPointLiteral`?

Reply:
Yorck probably did 16 months ago yeah ;) Fixed